transform_assumption.ml

```diff
@@ -32,11 +32,11 @@ let insert_checkpoints (func:afunction) =
           version=version.label;
           pos=checkpoint_label pc;
         } in
-        InsertBefore [Osr {label=checkpoint_label pc; cond=[]; target; map=osr};]
+        InsertBefore [Osr {label=checkpoint_label pc; cond=[]; target; var_map=osr; frame_maps=[]};]
     in
-    if pc = 0 then Unchanged else
     match[@warning "-4"] instrs.(pc) with
-    | Stop _ | Return _ | Label _ | Comment _ -> Unchanged
+    | Label _ | Comment _ ->
```

Comment:
To have more safepoints available. I noticed that in the case of:
```
call x = foo
return
```
there was no safepoint after the call. but we need one. Not sure if there was ever a good reason for the list.